examples/drivers/ZNB20.py

```diff
@@ -0,0 +1,1373 @@
+# Source: https://github.com/QCoherence/python_drivers/blob/master/ZNB20.py
+# -*- coding: utf-8 -*-
+# ZNB20.py is a driver for Rohde & Schwarz ZNB20 Vector Network Analyser
+# written by Thomas Weissl, modified by Nico Roch and Yuriy Krupko, 2014
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+
+from instrument import Instrument
+import qt
```

Comment:
Unless I misunderstand, qt is one heck of a big library to import. Why do we need this?